server/src/main/resources/org/cloudfoundry/identity/uaa/db/hsqldb/V4_7_0_1__Patch_Revocable_Tokens_To_Support_Long_Scopes.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE revocable_tokens ALTER COLUMN scope CLOB;
```

Comment:
question regarding file name: I've noticed we try to avoid version conflicts with upstream UAA in flyway scripts. Since they have a 4.7.0 release, is this considered a conflict?

Reply:
No since I added the _1 at the end. It is a micro micro version. Upstream has 4_7_0 then the next two are 4_7_1 and 4_7_2 so it will run right after 4_7_0 script.

Reply:
makes sense thanks